app/src/main/java/com/example/expensetracker/exp.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.expensetracker
+
+data class exp(val expense_title:String, val expense:Int, val `when`:String)
```

Comment:
Is it different from the entity? If not, you can use the entity class directly. No need to replicate the same code in multiple places.